### PR TITLE
Add Config command

### PR DESCRIPTION
**Pull request by Cikguseven**

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/LoadCommandTest.java [hunk 1]
```diff
@@ -100,4 +99,32 @@ public void equals() {
...
+
+        // different types -> returns false
+        assertFalse(loadFirstCommand.hashCode() == 1);
+
+        // null -> returns false
+        assertFalse(loadFirstCommand.hashCode() == 0);
+
+        // different student -> returns false
+        assertFalse(loadFirstCommand.hashCode() == loadSecondCommand.hashCode());
+    }
```
Comment (ChangruHenryQian) [inline src/test/java/seedu/address/logic/commands/LoadCommandTest.java:129]: Sufficient test cases to test hash code.

File: src/main/java/seedu/address/logic/commands/ConfigCommand.java [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        ConfigCommand e = (ConfigCommand) other;
+        return tutorialCount == e.tutorialCount
+                && assignmentCount == e.assignmentCount;
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(tutorialCount, assignmentCount);
+    }
+}
```
Comment (ChangruHenryQian) [inline src/main/java/seedu/address/logic/commands/ConfigCommand.java:103]: The readability of the code is increased by using proper line breaks.

**Review by ChangruHenryQian — COMMENTED** [review]
Overall, solid effort on the config command. Good job!
Just some minor things to fix to pass the Java CI.

**Comment by ngeeyonglim** [comment]
LGTM. Good code quality

**Pull request merged**